### PR TITLE
refactoring some instructions

### PR DESCRIPTION
**Pull request by choldgraf**
A PR to keep track of things that I run into as I go through the intro tutorial.

I also tried adding the README.rst into the index instead of having it as a separate page, LMK what folks think about that!

### CODE REVIEW

File: docs/troubleshooting.rst [hunk 1]
```diff
@@ -13,9 +13,12 @@ Troubleshooting
 Windows Issues
 --------------
 
-* Some people have reported issues using git bash; try using the Command Terminal instead.
+* Some people have reported issues using git bash; try using the
```
Comment (choldgraf) [inline docs/troubleshooting.rst]: here we could also recommend Windows Subsystem for Linux, which I've found quite useful!

**Comment by choldgraf** [comment]
ok I took a pass through the instructions and I think in general they look good, nice work @kysolvik! If you're fine w/ it, we can merge these changes (if you're happy w/ them) and keep track of to-do items in the issues.

File: docs/troubleshooting.rst (hunk 1 above)
Comment (kysolvik) [reply docs/troubleshooting.rst]: I've never used it but I've heard good things! Do you have an idea for sentence or two that would cover it?

**Comment by kysolvik** [comment]
this looks really great! Thanks @choldgraf, especially for the added language in the tutorial. Makes it much more clear what the different commands are doing

Do you have a suggestion for a sentence to cover the windows subsystem for linux? I've never used it. We could link to Microsoft's documentation, unless you have other ideas: https://docs.microsoft.com/en-us/windows/wsl/about

**Comment by choldgraf** [comment]
That probably what I'd do just - just something like:

```
If you're on Windows 10, you may also want to try `Windows Subsystem for Linux <https://docs.microsoft.com/en-us/windows/wsl/about>`_, which allows you to run a linux command line from within Windows. It is still somewhat experimental, but is useful for developing and testing code written in linux.
```

**Comment by kysolvik** [comment]
Great thanks! Merged your changes and I'll add the windows subsystem text

**Pull request merged**